assets/javascripts/discourse/controllers/s-show.js.es6

```diff
@@ -1,6 +1,7 @@
 import Controller from "@ember/controller";
```

Comment:
`s` is too obtuse for a controller name. You can call the controller/route/templates `subscriptions` and use `path` in the route map to call the route whatever you want if you want to simplify it.

Reply:
I agree here - it was something we inherited. I'll update this in an additional PR since we're already adding a lot here.